S/SuperLU_DIST/build_tarballs.jl

```diff
@@ -93,6 +93,9 @@ platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv6l" && libc(p
 platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && libc(p) == "musl"), platforms)
 platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && Sys.isfreebsd(p)), platforms)
 
+platforms = expand_gfortran_versions(platforms)
```

Comment:
Don't randomly expand libgfortran version if there's no need to.  This library doesn't even link to libgfortran

Reply:
ok, any suggestions on how to link it versus a more recent version of `MPItrampoline_jll`?

Reply:
No.  One (ideally not me, don't have much time) would have to investigate why dlopening fails.  #5028 is an example of process to track down these issues.

Reply:
thanks; I'm afraid this is above my skillset..